code/controllers/stonedmc/subsystem/direction.dm

```diff
@@ -9,15 +9,15 @@ SUBSYSTEM_DEF(direction)
 
 	// this is a two d list of defines to lists of mobs tracking that leader
 	// eg; list(CHARLIE_SL = list(<list of references to squad marines), XENO_NORMAL_QUEEN = list(<list of xeno mob refs))
-	var/list/processing_mobs = list()
+	var/list/list/processing_mobs = list()

```

Comment:
Is this intended?

Reply:
Yes.

Reply:
https://discordapp.com/channels/484170914754330625/484170915253321734/593717885872898049